node-bootstrapper/main.go

```diff
@@ -112,52 +85,25 @@ func Provision(ctx context.Context) error {
 		return errors.New("--provision-config is required")
```

Comment:
I believe if `provision-config` flag is not specified, then `provisionConfig` will return a pointer to the default value which in this case is empty string rather than returning `nil`. So we should check against empty string.